ibis/backends/duckdb/tests/test_client.py

```diff
@@ -420,3 +420,26 @@ def test_memtable_doesnt_leak(con, monkeypatch):
     df = ibis.memtable({"a": [1, 2, 3]}, name=name).execute()
     assert name not in con.list_tables()
     assert len(df) == 3
+
+
+def test_pyarrow_batches_chunk_size(con):  # 10443
+    import numpy as np
+
+    t = ibis.memtable(
+        {
+            "id": np.arange(500000),
```

Comment:
Can we make these smaller? I think the smallest chunk size duckdb can produce is 2048 (by default, and configurable only at compile time IIRC), so any multiple of that greater than 1 should be fine to test chunk size.

Reply:
Interesting, a chunk size of 800 worked. But we can indeed make these smaller!